### PR TITLE
Create changelog for 0.19.0

### PR DESCRIPTION
**Pull request by temanbrcom**
I have removed duplicated and internal changes, also the bug fixes for not released features. Please, check if missed something or removed by accident. The date specified in the changelog files is not agreed yet, so it may change.

### CODE REVIEW

**Comment by temanbrcom** [comment]
@Nurkambay 
@ap891843

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
+Changelog:
+
+## [0.19.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.18.0...0.19.0) (2021-06-08)
+
+#### Added
+* IDMS DML commands support
```
Comment (zeibura) [inline clients/cobol-lsp-vscode-extension/CHANGELOG.md:8]: this is listed in both 0.19.0 and 0.18.0. Should it be "expanded support for IDMS DML commands"? as in 0.19.0 supports more of them?

Comment (temanbrcom) [reply clients/cobol-lsp-vscode-extension/CHANGELOG.md:8]: Yeah, makes sense

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,39 @@
...
+* Expand support for IDMS DML commands [#704](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/704) ([e908c8f](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/e908c8f1466b0497fd8b238cdb193ad2d5947577)) ([286c42e](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/286c42ef50eb91da4aa826aba07a93158b1026df))  ([ca0cf91](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/ca0cf9120128f29892a226d2e587c0a098e31e74)) ([91646a4](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/91646a441ce94104c29c4a64820ef021f3d58072)) ([f55cdfc](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/f55cdfc059b4834c9e7a5c163e00a09947341827)) ([05020f9](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/05020f9d0b2558024fb8ce07885b6f5902aaf4f8))
+* add support for Datacom SQLDA and SQLCA implicit fields [#885](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/885) ([e6944d3](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/e6944d3340e65fa9f93a154c8f86ff92854c54e8))
```
Comment (grianbrcom) [inline CHANGELOG.md:26]: Don't forgot about https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/937

Comment (temanbrcom) [reply CHANGELOG.md:26]: Ok, I will add the commit mention here once the PR is merged.

**Review by zimlu02 — APPROVED** [review]

**Review by asatklichov — APPROVED** [review]
⛓️

**Review by zeibura — APPROVED** [review]

**Pull request merged**